.github/workflows/test.yml

```diff
@@ -49,6 +49,14 @@ jobs:
           conda list
           conda activate srbench
 
+      # This is useful for up-to-date CI, but
```

Comment:
I'm not too familiar with Julia, what is happening here? Is it over an above including Julia in `environment.yml`?

It it's going to potentially break CI when Julia packages update, i'd rather use a fixed version and update that as needed for srbench dependencies.

Reply:
This can be removed now. Basically, there is a problem with the Julia package servers in that sometimes they are up to a few days out-of-date. I included the most up-to-date PySR version in this PR which required this forced package server update. However, it's been a few days so the servers are all likely up-to-date!

Reply:
i sent a PR to your fork for this